lnbits/extensions/boltcards/crud.py

```diff
@@ -0,0 +1,143 @@
+from optparse import Option
+from typing import List, Optional, Union
+
+from lnbits.helpers import urlsafe_short_hash
+
+from . import db
+from .models import Card, CreateCardData, Hit
+
+
+async def create_card(data: CreateCardData, wallet_id: str) -> Card:
+    card_id = urlsafe_short_hash()
+    await db.execute(
+        """
+        INSERT INTO boltcards.cards (
+            id,
+            wallet,
+            card_name,
+            uid,
+            counter,
+            withdraw,
+            file_key,
+            meta_key
+        )
+        VALUES (?, ?, ?, ?, ?, ?, ?, ?)
+        """,
+        (
+            card_id,
+            wallet_id,
+            data.name,
```

Comment:
In my testing of this branch locally, I had to change `data.name` to `data.card_name`.

Reply:
Fixed. Thanks.